pkg/apis/core/helper/qos/qos.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// NOTE: DO NOT use those helper functions through client-go, the
+// Package qos provides helper functions related to QoS.
+// NOTE: DO NOT use these helper functions through client-go, the
 // package path will be changed in the future.
```

Comment:
can drop this NOTE, this package is not accessible from client-go

Reply:
Thanks for the feedback @liggitt! I'll fix this very soon